llpc/unittests/util/testUtil.cpp

```diff
@@ -0,0 +1,190 @@
+/*
+ ***********************************************************************************************************************
+ *
+ *  Copyright (c) 2021 Google LLC. All Rights Reserved.
+ *
+ *  Permission is hereby granted, free of charge, to any person obtaining a copy
+ *  of this software and associated documentation files (the "Software"), to deal
+ *  in the Software without restriction, including without limitation the rights
+ *  to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ *  copies of the Software, and to permit persons to whom the Software is
+ *  furnished to do so, subject to the following conditions:
+ *
+ *  The above copyright notice and this permission notice shall be included in all
+ *  copies or substantial portions of the Software.
+ *
+ *  THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ *  IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ *  FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ *  AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ *  LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ *  OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ *  SOFTWARE.
+ *
+ **********************************************************************************************************************/
+
+#include "llpcUtil.h"
+#include "vkgcDefs.h"
+#include "vkgcUtil.h"
+#include "lgc/EnumIterator.h"
+#include "gmock/gmock.h"
+#include "gtest/gtest.h"
+
+using namespace Vkgc;
+using namespace llvm;
+
+using ::testing::ElementsAre;
+
+namespace Llpc {
+namespace {
+
+// cppcheck-suppress syntaxError
+TEST(UtilTest, PlaceholderPass) {
+  EXPECT_TRUE(true);
```

Comment:
Do we need this placeholder? What's its purpose?

Reply:
I use as the simplest possible test that's trivially correct. Then when I suppress cppcheck errors it's clear that the cppcheck is at fault and not the test. But in this case, it does not appear to work because I had to annotate the next one too.

Reply:
I had to re-add it to make cppckeck happy